classes/global.html

```diff
@@ -7,19 +7,20 @@
 Code distributed by Google as part of the polymer project is also
 subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
 -->
+
 <link rel="import" href="../paper-styles-classes.html">
 
 <!--
-A set of base styles that are applied to the document and standard elements that
-match the Material Design spec.
+Note that this file probably doesn't do what you expect it to do. It's not
+a `<style is=custom-style include="..."` type of style include, which mean
+these styles will only apply to the main document, regardless of where
+you import this file.
+
+For a set of styles that can be applied to an element,
+check iron-demo-helpers/demo-pages-shared-styles.html.
```

Comment:
Oh, I think the first part of the comment makes sense. I'm wondering more about whether `demo-pages-shared-styles.html` is the right place to redirect to from this file since it doesn't seem to do the same things that this file was intended to do. Yes, `demo-pages-shared-styles.html` is 'a set of styles that can be applied to an element', but is it really the set of styles I'm looking for if I want to switch away from using this file?